Makefile

```diff
@@ -278,6 +280,10 @@ test-js-phantom: gui
 
 .PHONY: test-js-karma
 test-js-karma: gui
+	$(NODE_MODULES)/.bin/karma start karma.conf.js --single-run --browsers PhantomJS --no-colors --log-level warn --reporters mocha
```

Comment:
Some flags (`--no-colors`) were added to improve the CI build. Others (`--browsers`) were added so that we can add settings in the `karma.conf.js` without having to worry about impacting the CI build.